ci/container_scripts/install_deps.sh

```diff
@@ -12,9 +12,10 @@ APT_PACKAGES=(
   libglib2.0-0
   libglib2.0-dev
   make
+  netbase
```

Comment:
Add to the installation docs required dependencies?

Reply:
Done